protocols/Create_Callrate_file_PerSentrixBarcode.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+#MOLGENIS walltime=05:59:00 mem=10gb ppn=6
+
+#string Project
+#string pythonVersion
+#string beadArrayVersion
+#string bpmFile
+#string projectRawTmpDataDir
+#list Sample_ID
+#string SentrixBarcode_A
+#list SentrixPosition_A
+#string pipeline
+#string CallrateDir
+#string gapVersion
+#string logsDir
+#string intermediateDir
+
+#Function to check if array contains value
+array_contains () {
+	local array="$1[@]"
+	local seeking=$2
+	local in=1
+	for element in "${!array-}"; do
+		if [[ "$element" == "$seeking" ]]; then
+			in=0
+			break
+		fi
+	done
+	return $in
+}
+
+module load "${pythonVersion}"
+module load "${beadArrayVersion}"
+module load "${gapVersion}"
+module list
+
+set -e
+set -u
+
+mkdir -p "${CallrateDir}/"
+
+makeTmpDir "${CallrateDir}/"
+tmpCallrateDir="${MC_tmpFile}"
+
+
+INPUTARRAYS=()
+
+for array in "${SentrixBarcode_A}_${SentrixPosition_A[@]}"
+do
+	array_contains INPUTARRAYS "${array}" || INPUTARRAYS+=("$array")    # If GTCfile does not exist in array add it
+done
+
+
+if [ "${pipeline}" == 'research' ]
+then
+	for i in "${INPUTARRAYS[@]}"
+	do
+		python "${EBROOTGAP}/scripts/Make_Callrate_Report.py" "${bpmFile}" "${projectRawTmpDataDir}/${i}/" "${tmpCallrateDir}/${i}_callratedata_project.txt"
+	done
+
+
+	rm -f "${tmpCallrateDir}/callratedata_project.txt"
+	for j in ${INPUTARRAYS[@]}
```

Comment:
je gooit hierboven de callratedata_project.txt weg, en hier maak je files aan met ${j}_callratedata_project.txt, denk dat dat niet de bedoeling is